README.md

```diff
@@ -199,16 +199,20 @@ please DO NOT file a public issue, instead send your report privately to
 `security@coredns.io`. Security reports are greatly appreciated and we
 will publicly thank you for it.
 
+CoreDNS Team may disclose privately the security issue in order to organize the patch and its release.
 
-### Coordination of security releases with Kubernetes project
+If you lead a project that includes CoreDNS and need organizing controlled security releases when CoreDNS triggers a security release, 
+contact us at `security@coredns.io` in order to be added to our "Collaborating Projects" and have access to early security release information.
 
-As CoreDNS is now a feature part of Kubernetes, we are organizing our security release process 
-in a way that is compatible with [Kubernetes's security release process](https://github.com/kubernetes/sig-elease/blob/master/security-release-process-documentation/security-release-process.md)
+### Coordination of security releases with Collaborating Projects
 
-1. CoreDNS Team may disclose privately the security issue in order to organize the patch and its release.
-1. CoreDNS Team will inform Kubernete's project using kubernetes-security@googlegroups.com as early as possible and coordinate 
-with Kubernete's Team the upcoming security release timeline in such a way that Kubernetes 
+CoreDNS manage a list of "Collaborating Projects" that are assessed CoreDNS consumers and reliable secure projects. 
+That list aims to coordinate the security release in a controlled manner: 
+ 
+1. When a security issue is raised that triggers a CoreDNS code change and therefore a security release, 
+CoreDNS Team will inform privately each of the Collaborating Projects as early as possible, providing the timeline of CoreDNS release, 
```

Comment:
Who has a better name for "Collaborating Projects" ? (maybe included Security in that name ..) ?

Reply:
I think we should do the opposite and require any interested project to join (a new) mailing list

Reply:
I guess, in practical, what would happen is:
1- we receive a vulnerability issue somehow (incl email at security@coredns.io)
2- we assess if that needs a security release of CoreDNS
if YES:
3- we (CoreDNS Team) elaborate a timeline for release (I guess in a way that is compatible with subscribed projects)
4- we then inform the dl-security-release of the issue and our plan to remediate
5- I guess if that needs bargaining for the release timeframe, then we will be contacted by those projects that expect adaptation of the plan.
6- When plan is definitive, resend an email to advertise that plan and ETA release for CoreDNS.

Would that work ?